app/jinja_filters.py

```diff
@@ -765,9 +766,18 @@ def map_list_collector_config(
                     use_answer_label=True,
                     item_name=item_name,
                 )
-                row_items.extend(summary_row.itemsList)
-
-        rows.append({"itemsList": row_items})
+                row_items_related.extend(summary_row.itemsList)
+            # Again, List Collector Content list summaries don't use the row_item with "title" and are structured differently
```

Comment:
I'm not sure we need this comment since it kind of repeats the one above, was thinking some of these maybe more suited/better explained in docstrings, and might drive us to think of some things to refactor. Obviously not for this ticket though 👍

Reply:
It is pretty convoluted what happens here, I agree, docstring would be good, it's just that I didn't even look at the whole of this code (for this class) only the bottom part that was relevant for my task.

Reply:
Yeah I think this will need to be refactored at some point but as for this ticket its fine